package.json

```diff
@@ -51,7 +55,7 @@
     "nodemon": "^2.0.7",
     "prettier": "^2.2.1",
     "serverless": "^2.35.0",
-    "sinon": "^9.2.2",
+    "sinon": "^9.2.4",
```

Comment:
we're using jest as an assertion library, why introduce sinon ?

Reply:
It was already there, it's just the version number that is different.